doc/guides/dts/index.rst

```diff
@@ -71,11 +71,15 @@ The device tree files are compiled using the device tree compiler.  The compiler
 runs the .dts file through the C preprocessor to resolve any macro or #defines
 utilized in the file.  The output of the compile is another dts formatted file.
 
-After compilation, a python script extracts information from the compiled device
+After compilation, a Python script extracts information from the compiled device
 tree file using a set of rules specified in YAML binding files.  The extracted
 information is placed in a header file that is used by the rest of the code as
 the project is compiled.
 
+See the file ``dts/bindings/device_node.yaml.template`` for a binding template.
```

Comment:
You can create a clickable link to the file by writing this as:
```
:zephyr_file:`dts/bindings/device_node.yaml.template` 
```

Reply:
Fixed. I added links to `dts/bindings/` and `scripts/dts/` too.

Reply:
@dbkinder ping